### PR TITLE
Run CI jobs in parallel

### LINKED ISSUE #267: Separate the steps in CI to different jobs
Since the 3 different stacks can run independent of each other, it might be good to separate the run of hadoop, S3 and firebase messaging stacks into separate CI jobs, so these can be run in parallel and hence complete the CI on PRs faster.

### PR DESCRIPTION
**Pull request by baixiac**
Closes #267.

The temporary patch on `topic_init.sh` for kafka-init can be removed once a new image of `radarbase/radar-schemas-tools` is released with https://github.com/RADAR-base/RADAR-Schemas/pull/269.

### CODE REVIEW

File: dcompose-stack/radar-cp-s3-stack/docker-compose.yml [hunk 1]
```diff
@@ -264,10 +264,11 @@ services:
   #---------------------------------------------------------------------------#
   kafka-init:
     image: radarbase/radar-schemas-tools:${RADAR_SCHEMAS_VERSION}
+    restart: "no" # On start failure, users need to run "install" again
     networks:
       - kafka
       - zookeeper
-    command: "topic_init.sh"
+    command: "/schema/conf/topic_init.sh"
```
Comment (yatharthranjan) [inline dcompose-stack/radar-cp-s3-stack/docker-compose.yml:271]: The `topic_init` script is [copied](https://github.com/RADAR-base/RADAR-Schemas/blob/0a16c8059c801f05327e6b8d04eda9209af75f55/Dockerfile#L64) to the `/usr/bin` directory so should be available on `$PATH` and hence no need to specify an absolute path.

Comment (baixiac) [reply dcompose-stack/radar-cp-s3-stack/docker-compose.yml:271]: Yes. `/schema/conf/topic_init.sh` is a local copy/patch of the topic_init.sh of `radarbase/radar-schemas-tools`, on which the [required change](https://github.com/RADAR-base/RADAR-Schemas/pull/269) has not been released to docker hub yet.

Comment (baixiac) [reply dcompose-stack/radar-cp-s3-stack/docker-compose.yml:271]: Watching the Release 0.7.2: https://github.com/RADAR-base/RADAR-Schemas/pull/270

**Comment by baixiac** [comment]
Awaiting the new release of radar-schemas: https://github.com/RADAR-base/RADAR-Schemas/pull/279.

**Review by nivemaham — APPROVED** [review]
Lgtm

**Pull request merged**